--- Utilities/extra.cpp	2012-02-03 22:00:14.000000000 -0500
+++ Utilities/extra_new.cpp	2018-01-24 17:44:26.520000000 -0500
@@ -49,7 +49,7 @@
 
     ofstream out_Matrix (s_MatrixName.c_str());
     if(!out_Matrix) {
-	    cout<<"Error creating file: \""<<out_Matrix<<"\""<<endl;
+	    cout<<"Error creating file: \"" << s_MatrixName << "\""<<endl;
 	    exit(1);
     }
     
@@ -84,7 +84,7 @@
     string s_MatrixName = "pattern"+s_postfix+s_BaseName;
     ofstream out_Matrix (s_MatrixName.c_str());
     if(!out_Matrix) {
-	    cout<<"Error creating file: \""<<out_Matrix<<"\""<<endl;
+	    cout<<"Error creating file: \"" << s_MatrixName << "\""<<endl;
 	    exit(1);
     }
 
@@ -111,7 +111,7 @@
     string s_CompressedMatrixName = "CompressedMatrix"+s_postfix+s_BaseName;
     ofstream out_CompressedMatrix (s_CompressedMatrixName.c_str());
     if(!out_CompressedMatrix) {
-	    cout<<"Error creating file: \""<<out_CompressedMatrix<<"\""<<endl;
+	    cout<<"Error creating file: \"" << s_CompressedMatrixName << "\""<<endl;
 	    exit(1);
     }
     
@@ -140,7 +140,7 @@
     string s_MatrixName = "pattern_value"+s_postfix+s_BaseName;
     ofstream out_Matrix (s_MatrixName.c_str());
     if(!out_Matrix) {
-	    cout<<"Error creating file: \""<<out_Matrix<<"\""<<endl;
+	    cout<<"Error creating file: \"" << s_MatrixName << "\""<<endl;
 	    exit(1);
     }
 
@@ -167,7 +167,7 @@
     string s_CompressedMatrixName = "CompressedMatrix"+s_postfix+s_BaseName;
     ofstream out_CompressedMatrix (s_CompressedMatrixName.c_str());
     if(!out_CompressedMatrix) {
-	    cout<<"Error creating file: \""<<out_CompressedMatrix<<"\""<<endl;
+	    cout<<"Error creating file: \"" << s_CompressedMatrixName << "\""<<endl;
 	    exit(1);
     }
     
