diff --git Makefile Makefile
index 0ab9b9c..dca58f0 100644
--- Makefile
+++ Makefile
@@ -53,12 +53,14 @@ DMATRIX_DEPS_LIBS =	\
 PSOPT_DEPS_LIBS =	\
 	$(ADOLC_LIB)	\
 	-lipopt		\
-	-lcoinmetis	\
-	-lcoinmumps
+	-ldmumps	\
+	-lmumps_common	\
+	-lmpiseq	\
+	-lmetis
 
 SYS_LIBS =		\
 	-llapack	\
-	-lcblas		\
+	-lblas		\
 	-ldl		\
 	-lm
 
@@ -76,7 +78,7 @@ INCS =						\
 	-I$(IPOPT_DIR)/include/coin/ThirdParty	\
 	-I/usr/include/adolc
 
-FLAGS = -O0 -g -fomit-frame-pointer -fPIC
+FLAGS = -O0 -g -fomit-frame-pointer -fPIC -fopenmp
 
 WARNS =			\
 	-Wparentheses	\
